src/annotator.cpp

```diff
@@ -1156,200 +1155,259 @@ std::string Annotator::AnnotatorImpl::makeUniqueId()
+    case CellmlElementType::MAP_VARIABLES: {
         // Connections and map variables are not stored as a weak pointer so we can compare
         // shared pointers directly.
         itemsEqual = itemWeak->variablePair() == item->variablePair();
-    } else if ((type == CellmlElementType::ENCAPSULATION)
-               || (type == CellmlElementType::MODEL)) {
+        break;
+    }
+    case CellmlElementType::ENCAPSULATION:
+    case CellmlElementType::MODEL: {
         itemsEqual = equals(std::any_cast<ModelWeakPtr>(itemWeak->mPimpl->mItem),
                             std::any_cast<ModelWeakPtr>(item->mPimpl->mItem));
-    } else if (type == CellmlElementType::IMPORT) {
+        break;
+    }
+    case CellmlElementType::IMPORT: {
         itemsEqual = equals(std::any_cast<ImportSourceWeakPtr>(itemWeak->mPimpl->mItem),
                             std::any_cast<ImportSourceWeakPtr>(item->mPimpl->mItem));
-    } else if ((type == CellmlElementType::RESET)
-               || (type == CellmlElementType::RESET_VALUE)
-               || (type == CellmlElementType::TEST_VALUE)) {
+        break;
+    }
+    case CellmlElementType::RESET:
+    case CellmlElementType::RESET_VALUE:
+    case CellmlElementType::TEST_VALUE: {
         itemsEqual = equals(std::any_cast<ResetWeakPtr>(itemWeak->mPimpl->mItem),
                             std::any_cast<ResetWeakPtr>(item->mPimpl->mItem));
-    } else if (type == CellmlElementType::UNIT) {
+        break;
+    }
+    case CellmlElementType::UNIT: {
         // Unit is not actually stored as a weak pointer so we can compare shared pointers directly.
         itemsEqual = itemWeak->unitsItem() == itemShared->unitsItem();
-    } else if (type == CellmlElementType::UNITS) {
+        break;
+    }
+    case CellmlElementType::UNITS: {
         itemsEqual = equals(std::any_cast<UnitsWeakPtr>(itemWeak->mPimpl->mItem),
                             std::any_cast<UnitsWeakPtr>(item->mPimpl->mItem));
-    } else if (type == CellmlElementType::VARIABLE) {
+        break;
+    }
+    default: /* CellmlElementType::VARIABLE */
+    {
         itemsEqual = equals(std::any_cast<VariableWeakPtr>(itemWeak->mPimpl->mItem),
                             std::any_cast<VariableWeakPtr>(item->mPimpl->mItem));
+        break;
+    }
     }
+
     return itemsEqual;
 }
 
 bool Annotator::AnnotatorImpl::validItem(const AnyCellmlElementPtr &item)
 {
-    CellmlElementType type = item->type();
-    if ((type == CellmlElementType::COMPONENT)
-        || (type == CellmlElementType::COMPONENT_REF)) {
-        if (item->component() != nullptr) {
-            return true;
-        }
-    } else if ((type == CellmlElementType::CONNECTION)
-               || (type == CellmlElementType::MAP_VARIABLES)) {
+    bool result = false;
+    switch (item->type()) {
+    case CellmlElementType::COMPONENT:
+    case CellmlElementType::COMPONENT_REF: {
+        result = item->component() != nullptr;
+        break;
+    }
```

Comment:
No need for the curly brackets. Same for some other `case`s below.

Reply:
[67a06cb](https://github.com/cellml/libcellml/pull/1127/commits/67a06cbf036b01be249f6e6bdf442888aeb5ad8d)